<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>templates</class>
 <widget class="QWidget" name="templates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模板</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_15">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>新增</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>模板</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,0">
                 <item>
                  <widget class="QTableWidget" name="tableWidget">
                   <property name="selectionMode">
                    <enum>QAbstractItemView::NoSelection</enum>
                   </property>
                   <property name="columnCount">
                    <number>0</number>
                   </property>
                   <attribute name="horizontalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                     <layout class="QVBoxLayout" name="verticalLayout_3">
                      <item>
                       <widget class="QLabel" name="label_18">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                         <widget class="QLabel" name="label">
                          <property name="maximumSize">
                           <size>
                            <width>200</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>原数据字节开始位置</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEdit">
                          <property name="maximumSize">
                           <size>
                            <width>50</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                         <widget class="QLabel" name="label_3">
                          <property name="maximumSize">
                           <size>
                            <width>200</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>原数据字节结束位置</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEdit_2">
                          <property name="maximumSize">
                           <size>
                            <width>50</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                         <widget class="QLabel" name="label_2">
                          <property name="maximumSize">
                           <size>
                            <width>200</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>模板数据字节开始位置</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEdit_3">
                          <property name="maximumSize">
                           <size>
                            <width>50</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QLabel" name="label_4">
                          <property name="maximumSize">
                           <size>
                            <width>200</width>
                            <height>200</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>模板数据字节结束位置</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEdit_4">
                          <property name="maximumSize">
                           <size>
                            <width>50</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_17">
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                         <widget class="QPushButton" name="pushButton">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>添加</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="pushButton_2">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>删除</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_20">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_17">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>名称：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_11">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>定义：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_14"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <layout class="QGridLayout" name="gridLayout_8">
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="gridLayout_7">
                      <item row="0" column="0">
                       <widget class="QCheckBox" name="checkBox">
                        <property name="text">
                         <string>校验</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QWidget" name="widget_6" native="true">
                        <layout class="QGridLayout" name="gridLayout_6">
                         <item row="0" column="0">
                          <layout class="QGridLayout" name="gridLayout_5">
                           <item row="0" column="0">
                            <widget class="QRadioButton" name="radioButton">
                             <property name="text">
                              <string>和校验</string>
                             </property>
                            </widget>
                           </item>
                           <item row="0" column="1" colspan="2">
                            <widget class="QLabel" name="label_5">
                             <property name="text">
                              <string>开始位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="0" column="3">
                            <widget class="QLineEdit" name="lineEdit_5"/>
                           </item>
                           <item row="0" column="4">
                            <widget class="QLabel" name="label_7">
                             <property name="text">
                              <string>结束位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="0" column="5">
                            <widget class="QLineEdit" name="lineEdit_6"/>
                           </item>
                           <item row="0" column="6">
                            <widget class="QLabel" name="label_11">
                             <property name="text">
                              <string>所在位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="0" column="7">
                            <widget class="QLineEdit" name="lineEdit_12"/>
                           </item>
                           <item row="1" column="0" colspan="2">
                            <widget class="QRadioButton" name="radioButton_2">
                             <property name="text">
                              <string>CRC校验</string>
                             </property>
                            </widget>
                           </item>
                           <item row="1" column="2">
                            <widget class="QLabel" name="label_6">
                             <property name="text">
                              <string>开始位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="1" column="3">
                            <widget class="QLineEdit" name="lineEdit_7"/>
                           </item>
                           <item row="1" column="4">
                            <widget class="QLabel" name="label_8">
                             <property name="text">
                              <string>结束位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="1" column="5">
                            <widget class="QLineEdit" name="lineEdit_8"/>
                           </item>
                           <item row="1" column="6">
                            <widget class="QLabel" name="label_14">
                             <property name="text">
                              <string>所在位：</string>
                             </property>
                            </widget>
                           </item>
                           <item row="1" column="7">
                            <widget class="QLineEdit" name="lineEdit_13"/>
                           </item>
                           <item row="1" column="8">
                            <widget class="QLabel" name="label_15">
                             <property name="text">
                              <string>字节数</string>
                             </property>
                            </widget>
                           </item>
                           <item row="1" column="9">
                            <widget class="QLineEdit" name="lineEdit_15"/>
                           </item>
                          </layout>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QGridLayout" name="gridLayout_10">
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,0,1,3">
                      <item row="0" column="0">
                       <widget class="QCheckBox" name="checkBox_2">
                        <property name="maximumSize">
                         <size>
                          <width>1677215</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>帧计数</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                         <widget class="QLabel" name="label_9">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>所在位：</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="label_10">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="text">
                           <string>字节数：</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item row="0" column="2">
                       <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                         <widget class="QLineEdit" name="lineEdit_9">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEdit_10">
                          <property name="maximumSize">
                           <size>
                            <width>100</width>
                            <height>16777215</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item row="0" column="3">
                       <widget class="QLabel" name="label_16">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action">
   <property name="text">
    <string>新增</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
